### PR TITLE
Reduce number of PWA icons, fix paths

### LINKED ISSUE #124: [Bug] PWA not working in Firefox
I'm trying to save the app as a PWA on my Pixel 3A XL, running Android 10 August security patch, using Firefox 68.11.0.

The PWA save icon shows up at the top of the browser as expected. Tapping it shows the save confirmation pop-up, however the Homer icon is missing. Tapping Save then does nothing except close the pop-up.

I'm happy to provide screenshots on request, however I suspect this will be easy to reproduce.

Edit: I'm not a frontend developer, but I have encountered issues with PWAs like this before, and it had something to do with the Vue configuration. Maybe we start by checking that?

### PR DESCRIPTION
**Pull request by NotWoods**
## Description

Rather than supplying a bunch of icon variations, its a lot easier to just have 512x512 icons and let other platforms resize the icon.

Maskable icons and iOS icons are now set up properly with correct sizing and a background. [Preview here](https://maskable.app/?demo=https://raw.githubusercontent.com/NotWoods/homer/pwa-icons/public/assets/icons/icon-maskable.png).

Also added a monochrome icon by reusing the Safari Pinned Tab icon.

Paths are resolved relative to the manifest URL.

Fixes #124, #126

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I read & comply with the [contributing guidelines](https://github.com/bastienwirtz/homer/blob/master/CONTRIBUTING.md)
- [x] I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers. 
- [ ] I have made corresponding changes the documentation (README.md).
- [x] I've check my modifications for any breaking change, especially in the `config.yml` file

### CODE REVIEW

**Comment by NotWoods** [comment]
Hi @bastienwirtz, please take a look when you have a chance. Happy to answer any questions and make changes.

**Comment by bastienwirtz** [comment]
@NotWoods Looking at it right now! Thanks for your contribution! 
I'm gonna do some test, just to make sure the path work either in docker or direct. 

I know there is a lot of icons, but those can be auto generated as described here: https://github.com/bastienwirtz/homer/blob/master/docs/configuration.md#pwa-icons. 
On the other hand, it's kinda useless to have all individual size, I like your approach, it's more simple :thinking:

File: vue.config.js [hunk 1]
```diff
@@ -1,103 +1,65 @@
...
-      .loader("raw-loader")
+      .use('raw-loader')
+      .loader('raw-loader')
       .end();
   },
-  publicPath: "",
+  publicPath: '',
   pwa: {
-    manifestPath: "assets/manifest.json",
+    manifestPath: 'assets/manifest.json',
```
Comment (bastienwirtz) [inline vue.config.js:12]: :thinking: I think we could actually remove all the pwa configuration except from the manifestPath (used to inject the path in the header). 
The goal is to use the static manifest in the asset directory so anybody can customise the app name & other thing easily. What do you think?

**Review by bastienwirtz — CHANGES_REQUESTED** [review]
I'm in to reduce the number of icons. I just suggested a few modifications, let me know what you think. 

Thanks again !

File: vue.config.js (hunk 1 above)
Comment (NotWoods) [reply vue.config.js:12]: [@vue/cli-plugin-pwa](https://cli.vuejs.org/core-plugins/pwa.html#configuration) looks like it generates the manifest file unless you put an absolute path in there. I'm not 100% sure, but its possible the JSON file isn't even used right now.

**Comment by NotWoods** [comment]
I tweaked the Vue config so it pulls all values from the static JSON file. However, it doesn't look like you can turn off the manifest generation without using an absolute URL. I'll file a issue with Vue for that.

File: vue.config.js [hunk 2]
```diff
@@ -10,50 +10,11 @@ module.exports = {
...
-      },
-      {
-        src: "./icons/safari-pinned-tab.svg",
-        sizes: "any",
-        type: "image/svg+xml",
-        purpose: "monochrome",
-      },
-    ],
+    themeColor: "#3367D6",
+    manifestOptions: require('./public/assets/manifest.json'),
```
Comment (bwirtz) [inline vue.config.js]: Good idea!

**Review by bastienwirtz — APPROVED** [review]
Love idea to just require the static manifest, nicely done! Thanks!

**Pull request merged**